src/ORM/Query/SelectQuery.php

```diff
@@ -760,16 +759,6 @@ protected function _decorateResults(iterable $result): ResultSetInterface
         return $result;
     }
 
-    /**
-     * Returns the name of the class to be used for decorating results
-     *
-     * @return class-string<\Cake\Datasource\ResultSetInterface>
-     */
-    protected function _decoratorClass(): string
```

Comment:
Is this a method that is worth mentioning as a behavior change in the upgrade guide?

Reply:
It would be of interest only to someone who has overridden this class. Anyone smart enough to use a customized query class wouldn't have any problem making required adjustments to their code due to this change :)